src/main/java/Event.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Event task which inherits from Task.
+ * Adds the ability to have a timing (at).
+ */
+public class Event extends Task {
+    String at;
```

Comment:
Perhaps you can specify whether this is a "public", "protected" or "private" member?